packages/daisyui/src/types.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module '*.css?inline' {
+    const content: string;
```

Comment:
Hmmm, I'm surprised the linter didn't change the indents to 2 spaces

Reply:
Agreed. It's forcing to 4-spaces here. Will make two.